package.json

```diff
@@ -47,8 +47,10 @@
     "ms-uk-local-authorities": "^2.1.0",
     "ms-uk-police-forces": "^2.0.0",
     "ms-uk-regions": "^2.1.1",
+    "node-sass": "^4.13.0",
```

Comment:
Where's this dependency come from? Nothing else in this PR involves SASS.

Reply:
May have to separate this out.
I have node 12 installed on this new machine so when I ran the app it required that I download the updated version.

Cheers

Reply:
seems a bit weird to have to have this

Reply:
Yeah I'll take this out. It updated on my end, no need unless our node version is going to be upgraded to 12. In that case it should be a separate PR.

Reply:
yeah and probably separate PR as well if that's the case